multi_localization/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2016 Taŭga Tecnologia - Aristides Caldeira <aristides.caldeira@tauga.com.br>
+# License AGPL-3 or later (http://www.gnu.org/licenses/agpl)
+#
+
+{
+    'name': u'Multi localization',
+    'version': '10.0.0.0.0',
```

Comment:
You can see all the OCA Guidelines here:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers